modules/balana-core/src/main/java/org/wso2/balana/xacml3/Attributes.java

```diff
@@ -210,9 +210,9 @@ public void encode(StringBuilder builder) {
         builder.append("<Attributes Category=\"").append(category.toString()).append("\">");
 
         for(Attribute attribute : attributes){
-            if(attribute.isIncludeInResult()){
-                attribute.encode(builder);
-            }
+            //if(attribute.isIncludeInResult()){
```

Comment:
Looks like a redundant check. Let's get rid of the commented if check to keep your fix clean.